Doc/c-api/init.rst

```diff
@@ -1188,18 +1204,16 @@ code, or when embedding the Python interpreter:
 
 .. c:function:: PyThreadState* PyEval_SaveThread()
 
-   Release the global interpreter lock (if it has been created) and reset the
-   thread state to ``NULL``, returning the previous thread state (which is not
-   ``NULL``).  If the lock has been created, the current thread must have
-   acquired it.
```

Comment:
"If the lock has been created": it's ok to remove this sentence, the GIL is always created a startup for a long time.